### ISSUE TITLE
The two email accounts are the same

### ISSUE THREAD

**Issue by Hualoong** [Tom60chat/Microsoft-Rewards-Farmer-Sharp#98]
The two email accounts are the same, but the suffixes are different. You can only use one of them. The preliminary troubleshooting should be the corresponding account problem under the Sessions directory.

**Comment by Tom60chat** [comment]
Hi @hualoong
I don't really understand what you are saying, I guess you used a translator.

What you mean is that you can't use two emails like :
- name@domaine.com
- name@anotherDomaine.com

It's true that I hadn't thought of that.
This need to be fixed

**Comment by pearlheartgtr** [comment]
Yes! I had the same problem.  Mine were with a hotmail and a yahoo address. I ended up copying the main folder and renaming the new one. I set up the first with 3 accounts (hotmail) and the second with the account that I was having trouble with (yahoo). But, here's the thing, leave the one that is actually working in the first folder (mine was hotmail) and the second one that isn't showing up (yahoo) in the other.

**Comment by Tom60chat** [comment]
Yes, I see.

I know why this append, because I just don't check everythings after the @.

Should be an easy fix.
